src/UI/api/apollo.ts

```diff
@@ -1,18 +1,18 @@
-import { ApolloClient, DefaultOptions, InMemoryCache } from "@apollo/client";
+import { ApolloClient, DefaultOptions, InMemoryCache } from '@apollo/client';
 
 const defaultOptions: DefaultOptions = {
-    watchQuery: {
-      fetchPolicy: 'no-cache',
-    },
-    query: {
-      fetchPolicy: 'no-cache',
-    },
-  }
+  watchQuery: {
+    fetchPolicy: 'no-cache',
+  },
+  query: {
+    fetchPolicy: 'no-cache',
+  },
+};
 
 const client = new ApolloClient({
-    uri: "/vector-api/graphql",
-    cache: new InMemoryCache(),
-    defaultOptions: defaultOptions,
+  uri: '/vector-api/graphql',
+  cache: new InMemoryCache(),
+  defaultOptions: defaultOptions,
 });
 
-export default client;
\ No newline at end of file
+export default client;
```

Comment:
This file needs to be removed - we're not using apollo anymore

Reply:
I am unable to remove it without the site breaking